.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2.1
+
+jobs:
+  push:
+    machine: true
```

Comment:
Can we just use a Docker image instead of `machine: true`? If you're looking for a minimal config to pull from, see https://github.com/stackrox/clairify/blob/master/.circleci/config.yml